src/helm.exec.ts

```diff
@@ -76,6 +76,10 @@ export function helmDepUp() {
 }
 
 export function helmCreate() {
+    if (!vscode.workspace.rootPath) {
```

Comment:
The API `vscode.workspace.rootPath` is [deprecated](https://github.com/Microsoft/vscode/issues/39625) since vscode added the multiple workspace support feature. Need use `vscode.workspace.workspaceFolders` instead. And we have similar workspace picking up or empty check issue for many commands, maybe we could add an utility method for workspace check/pickup.

At the `extension.ts` file, i have implemented a method for that, suggest to move it to `extensionUtils.ts` file for sharing.

```
async function showWorkspaceFolderPick(): Promise<vscode.WorkspaceFolder> {
    if (!vscode.workspace.workspaceFolders) {
        vscode.window.showErrorMessage('This command requires an open folder.');
        return undefined;
    } else if (vscode.workspace.workspaceFolders.length === 1) {
        return vscode.workspace.workspaceFolders[0];
    }
    return await vscode.window.showWorkspaceFolderPick();
}
```

Reply:
Perfect, done and ready for re-review.